aeternity/config.go

```diff
@@ -97,44 +149,43 @@ var Config = ProfileConfig{
 		URLChannels: "https://sdk-mainnet.aepps.com",
 		NetworkID:   "ae_mainnet",
 	},
+	Compiler: CompilerConfig{
+		URL:     "http://localhost:3080",
+		Backend: CompilerBackendAEVM,
+	},
 	Client: ClientConfig{
 		BaseGas:    *utils.NewIntFromUint64(15000),
 		GasPerByte: *utils.NewIntFromUint64(20),
-		GasPrice:   *utils.NewIntFromUint64(1000000000),
+		GasPrice:   *utils.NewIntFromUint64(1e9),
 		TTL:        500,
 		Fee:        *utils.RequireIntFromString("200000000000000"),
```

Comment:
why the fee is set here? isn't it automatically calculated?

Reply:
It's not calculated until you tell it to. If you can't be bothered to call EstimateFee(), you just use this default value which should be big enough for most purposes.